notebooks/name-entity-extraction.ipynb

```diff
@@ -0,0 +1,452 @@
+{
```

Comment:
<blockquote>Using them, we will experiment with other models and applications which are just as responsive as the first, but more directed for specific purposes.</blockquote><p>A bit too general, Can you say which specific purposes?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/graphcore/optimum-graphcore-fork/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>